src/Pyz/Yves/CartPage/Theme/default/components/organisms/cart-sidebar/cart-sidebar.twig

```diff
@@ -182,10 +182,14 @@
             modifiers: ['summary'],
         } only %}
             {% block body %}
+                {% set cartItemsCount = 0 %}
+                {% for cartItem in data.transformedCartItems %}
+                    {% set cartItemsCount = cartItemsCount + cartItem.quantity %}
+                {% endfor %}
```

Comment:
The `data.transformedCartItems` uses only for the counter.
So it should be prepared outside instead.